cirq-core/cirq/devices/named_topologies_test.py

```diff
@@ -85,6 +85,9 @@ def test_line_topology():
     assert LineTopology(2).n_nodes == 2
     assert LineTopology(2).graph.number_of_nodes() == 2
 
+    mapping = topo.nodes_to_linequbits(offset=3)
+    assert all(isinstance(q, cirq.LineQubit) and q >= cirq.LineQubit(0) for q in mapping.values())
```

Comment:
The test does not check keys in the mappings dictionary.  It will also pass if values were generated with offset=2 instead of 3; please adjust so it only passes when the offset is exactly 3.

Also it is better to check only one condition per assert statement so this should rather be split to two assertions for (i) the type and (ii) specific values of the LineQubits in the mapping dictionary.